app/login.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { View, TextInput, Button, Text, StyleSheet } from 'react-native';
+
+const LoginPage = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
```

Comment:
Overvej at bruge et objekt så at formen er repræsenteret som et objekt med sername og password. Det sørger for at vi undgår unødvendige renderinger.